unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -551,5 +551,16 @@ public static bool TryFindChildRecursively(this Transform transform, string name
             }
             return false;
         }
+
+        public static bool IsPointerOverUIElement(Vector3 mousePosition)
+        {
+            var eventData = new PointerEventData(EventSystem.current);
+            eventData.position = mousePosition;
+            var results = new List<RaycastResult>();
+            EventSystem.current.RaycastAll(eventData, results);
+            return results.Count > 1;
+        }
```

Comment:
I have found this same `IsPointerOverUIElement` function in several places in our project (for example in `BIWUtils`)... maybe it is a good moment to centralize it in only one place and avoid to duplicate code.

Reply:
> I left a suggestions in the code review.
> 
> In other hand, I don't know wy but the use of +/- keys is not working for me (I only can apply soom in/out with the mouse wheel).

@sandrade-dcl have you tried with both your +/- keys on the regular keyboard and also on the numpad (if you have it)?